### ISSUE TITLE
environmentbase create explodes due to missing global.valid_regions

### ISSUE THREAD

**Issue by adimarco** [DualSpark/cloudformation-environmentbase#47]
Walking through the setup for a new environment with a fresh checkout gets:

```
$ environmentbase create

ERROR:
    Config file missing section global.valid_regions

Try running with the --debug flag
```

**Comment by sesas** [comment]
Hi @adimarco, 

What version was of envbase was that? 
Could you re-run it with the `--debug` and paste here the traceback? 

For reference, this commit should have removed the need for that global.valid_regions 8f4222146e26c38a7c936c3467344d61126d9b80

WRT to PR https://github.com/DualSpark/cloudformation-environmentbase/pull/48, I think we should work toward removing `valid_regions` rather than putting it back.

**Comment by sesas** [comment]
This issue should be fixed now w/ 45633b1. 

I tried it on local, let me know if you still see the problem.  If not we can close this issue.
